### ISSUE TITLE
Loading disk DataSets with absolute path as URI fails under Windows

### ISSUE THREAD

**Issue by mrmh2** [jic-dtool/dtoolcore#15]
The following test fails under Windows:

```
from . import TEST_SAMPLE_DATA

def test_abspath_uri():
    from dtoolcore import DataSet

    ds = DataSet.from_uri(TEST_SAMPLE_DATA)
```

with:

```
>       StorageBroker = storage_broker_lookup[parsed_uri.scheme]
E       KeyError: 'c'

dtoolcore\__init__.py:54: KeyError
```

**Comment by mrmh2** [comment]
This is because Windows, URIs and Python are kind of horrible:

```
In [42]: os.path.isdir("C:\windows")
Out[42]: True

In [43]: pathlib.Path("C:\windows").as_uri()
Out[43]: 'file:///C:/windows'

In [46]: urllib.parse.urlparse('file:///C:/windows')
Out[46]: ParseResult(scheme='file', netloc='', path='/C:/windows', params='', query='', fragment='')

In [47]: os.path.isdir('/C:/windows')
Out[47]: False
```

This is probably urllib.parse.urlparse's fault?

**Comment by tjelvar-olsson** [comment]
Fixed in b0c7408
